FlowersStore/Utils/Layout/UIView+Constraints.swift

```diff
@@ -0,0 +1,212 @@
+            self
+                .topAnchor(equalTo: view.topAnchor, constant: constant)
+                .leadingAnchor(equalTo: view.leadingAnchor, constant: constant)
+                .trailingAnchor(equalTo: view.trailingAnchor, constant: -constant)
+                .bottomAnchor(equalTo: view.bottomAnchor, constant: -constant)
+        }
+    }
+
+    /// Function to add more than one view to view's hierarchy
+    func addSubviews(_ views: UIView...) {
+        views.forEach { addSubview($0) }
+    }
+
+    @discardableResult func topAnchor(equalTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = topAnchor.constraint(equalTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func topAnchor(greaterThan anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = topAnchor.constraint(greaterThanOrEqualTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func topAnchor(lessThanOrEqualTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = topAnchor.constraint(lessThanOrEqualTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func topAnchor(greaterThanOrEqualTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = topAnchor.constraint(greaterThanOrEqualTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func topAnchorToSuperview(constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        guard let superview = superview else { fatalError("View doesn't have a superview") }
+        let constraint = topAnchor.constraint(equalTo: superview.topAnchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func bottomAnchor(equalTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = bottomAnchor.constraint(equalTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func bottomAnchor(greaterThanOrEqualTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'bottomA...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Utils/Layout/UIView+Constraints.swift;
  Line: 71;
-->

- :warning: Function 'bottomAnchor(greaterThanOrEqualTo:constant:priority:)' is unused